src/App.js

```diff
@@ -1,14 +1,65 @@
-import React from "react";
+import { Component } from "react";
 import "./App.css";
 
-class App extends React.Component {
-  render() {
-    return (
-      <main>
-        <p>React State Lab</p>
-      </main>
-    );
+class App extends Component {
+  constructor(){
+    super()
+
+    this.state = {
+      currentScore: 0,
+      incrementor: 1,
+      // multiplier: incrementor +1,
+    };
+  }
+
+  increaseScore = () => {
+    const { currentScore, incrementor } = this.state;
+    this.setState({
+      currentScore: currentScore + incrementor
+    })
+  } 
+
+  levelUp = () => {
+    const {currentScore, incrementor} = this.state;
+      if(currentScore < 10) {
+        alert("You can't afford that!")
+      }else{
+        this.setState({
+          currentScore: currentScore -10,
+          incrementor: incrementor + 1
+        })
+      };
+  }
+
+  handleReset = () => {
+    this.setState({
+      currentScore: 0,
+      incrementor: 1,
+    });
+  }
+
+render() {
+    let {currentScore, incrementor} = this.state;
+      if(currentScore < 100) {
+        return (
+        <main>
+        <h1>Current Score: {currentScore}</h1>
+        <button onClick={this.increaseScore}>+{incrementor}</button>
+        <button onCLick={this.levelUp}>Pay 10 points to change from +{incrementor} to +{incrementor + 1}</button>
```

Comment:
```suggestion
        <button onClick={this.levelUp}>Pay 10 points to change from +{incrementor} to +{incrementor + 1}</button>
```
The typo on this line is the _only_ reason your tests weren't passing. `onClick` is a valid React event. "onC**L**ick" with a capital "L" is **not**. Sometimes it's the tiniest syntax errors/typos! 

The way I was able to troubleshoot this (and you can try it too for future bugs!) was by seeing that clicking this button wasn't doing anything, even if i console logged "hi" inside of the levelUp function. This made me realize that the onClick event wasn't registering at all, which is what led me to this particular line of code where the typo was.